docs/developer_guidelines.md

```diff
@@ -124,68 +125,79 @@ Resources:
 - [One Idea is One Commit] - Interesting committing strategy, comparable to
   atomic commits.
 
-Use the `gitlint` tool to check your commit messages. See
-[tools/gitlint/README.md] for more information.
+### Message style and convention
 
-### Message style
+We follow [Conventional Commits] with a minor deviation:
 
-There are two possible styles for the commit messages: short and full.
+- since we do not usually create libraries, everything related to the
+  `BREAKING-CHANGE` and `!` rule is ignored. Specifically ignore rules 11., 12.,
+  13., 15., and 16. of the Conventional Commits [specification].
 
-#### Short
+[specification]: https://www.conventionalcommits.org/en/v1.0.0/#specification
 
-Short commit messages should be around 50 characters long, the maximum is 72.
+We use [`commited`] tool (run by the `pre-commit`) to validate commit messages.
+For relevant configuration see [`commited.toml`] file in our template
```

Comment:
the link doesn't work

Reply:
Correct, as it is not yet on that location (PR is in review). To check it see this PR: https://github.com/IRNAS/irnas-zephyr-template/pull/20